src/qt/rpcconsole.cpp

```diff
@@ -958,6 +959,18 @@ void RPCConsole::updateNetworkState()
     }
 
     ui->numberOfConnections->setText(connections);
+
+    // localAddressess
+    std::string localAddresses="";
```

Comment:
you could just make this a QString, then `item.first.ToString()` -> `QString::fromStdString(item.first.ToString())`

Reply:
Should rename from `localAddresses` -> `local_addresses` to keep in line with our coding style